lib/migration.js

```diff
@@ -1022,14 +1022,16 @@ function mixinMigration(MySQL, mysql) {
     return columnType;
   }
   function expectedColNameForModel(propName, modelToCheck) {
-    var mysql = modelToCheck.properties[propName].mysql;
-    if (typeof mysql === 'undefined') {
-      return propName;
-    }
-    var colName = mysql.columnName;
-    if (typeof colName === 'undefined') {
-      return propName;
+    if (modelToCheck.properties[propName].mysql) {
+      var mysql = modelToCheck.properties[propName].mysql;
+      if (typeof mysql === 'undefined') {
```

Comment:
Can we keep the original style please? Also we use `const` instead of var in new code.

```suggestion
    const mysql = modelToCheck.properties[propName].mysql;
    if (typeof mysql === 'undefined') {
```